platform/darwin/test/MGLDocumentationExampleTests.swift

```diff
@@ -47,6 +47,18 @@ class MGLDocumentationExampleTests: XCTestCase, MGLMapViewDelegate {
         styleLoadingExpectation.fulfill()
     }
     
+    func testMGLLight() {
+        //#-example-code
+        light = MGLLight()
+        let position = MGLSphericalPositionMake(5, 180, 80)
```

Comment:
Use the Swift struct initializer `MGLSphericalPosition(radial:azimuthal:polar:)` instead of the C-style factory function `MGLSphericalPositionMake(_:_:_:)`.